crates/biome_service/src/workspace/server.rs

```diff
@@ -49,8 +50,8 @@ pub(super) struct WorkspaceServer {
     /// features available throughout the application
     features: Features,
 
-    /// global settings object for this workspace
-    settings: Projects,
+    /// projects held by the workspace
+    projects: Projects,
```

Comment:
Hah, I did the exact same rename in my PR. No worries, it should be a very minor conflict for the last one to merge :)